cl/search/views.py

```diff
@@ -1,7 +1,10 @@
 import logging
+import math
```

Comment:
I think we can remove this one since it’s not being used, as well as `import time` below.